drivers/gpu/drm/rp1/rp1-dsi/rp1_dsi_dsi.c

```diff
@@ -1471,6 +1516,28 @@ void rp1dsi_dsi_send(struct rp1_dsi *dsi, u32 hdr, int len, const u8 *buf)
 		usleep_range(100, 150);
 	}
 
+	/*
+	 * Update global configuration flags for LP/HS and ACK options.
+	 * XXX It's not clear if having empty FIFOs (checked above and below) guarantees that
+	 * the last command has completed and been ACKed, or how closely these control registers
+	 * align with command/payload FIFO writes (as each is an independent clock-crossing)?
+	 */
+	if (use_lpm != dsi->using_lpm || req_ack != dsi->using_ack) {
```

Comment:
Do we have to retain the state and avoid rewriting the registers if unchanged? What's the downside of making it unconditional?

Reply:
Only a slight speed penalty.

Reply:
It's actually far from clear that it's safe to change these registers while running, although the Linux API seems to require it. That said, writing them with the _same_ values again is safe enough so that is no reason not to do so unconditionally.